clash-prelude/src/Clash/Sized/Internal/BitVector.hs

```diff
@@ -81,6 +83,7 @@ module Clash.Sized.Internal.BitVector
   , gt#
   , le#
     -- *** Enum (not synthesizable)
+  , toEnum#
```

Comment:
The heading says _not synthesizable_, but `toEnum#` is synthesizable, right? And shouldn't `fromEnum#` be exported as well?

This same thing applies to all following added `toEnum#` and `fromEnum#` functions.

Reply:
Yes, and yes. I'm still debugging the testsuite failures though right now :upside_down_face:

Reply:
I'm not at my sharpest, I kinda missed the _Draft_ status :-).